test/src/Admin/Model/DoctrineAutodiscoveryModelTest.php

```diff
@@ -26,15 +27,19 @@ public function testORMAutodiscoveryEntitiesWithFields()
         $result = $model->fetchFields(null, null, 'doctrine.entitymanager.orm_default');
 
         $this->assertInternalType('array', $result);
-        $this->assertCount(2, $result);
+        $this->assertCount(3, $result);
 
         $this->assertEquals(Album::class, $result[0]['entity_class']);
         $this->assertEquals('Album', $result[0]['service_name']);
         $this->assertCount(2, $result[0]['fields']);
 
-        $this->assertEquals(Artist::class, $result[1]['entity_class']);
-        $this->assertEquals('Artist', $result[1]['service_name']);
-        $this->assertCount(2, $result[1]['fields']);
+        $this->assertEquals(Product::class, $result[1]['entity_class']);
+        $this->assertEquals('Product', $result[1]['service_name']);
+        $this->assertCount(1, $result[1]['fields']);
+
+        $this->assertEquals(Artist::class, $result[2]['entity_class']);
+        $this->assertEquals('Artist', $result[2]['service_name']);
+        $this->assertCount(2, $result[2]['fields']);
```

Comment:
I don't understand why this test changes at this time; could you explain why the result now contains 3 entities, and the `Product` entity is now present?

Reply:
I've added new entity Product with field "version", so now we have 3 entities with fields, not 2 as before.